upgrading/index.adoc

```diff
@@ -81,6 +81,19 @@ in-place upgrades or blue-green deployments.
 The manual upgrade method breaks down the steps that happen during an automated
 Ansible-based upgrade and provides the equivalent commands to run manually. Using this method describes the in-place xref:install-config-upgrading-strategies[upgrade strategy].
 
+[[upgrading-swap-memory]]
+== Considerations for upgrading with swap memory
+
+A clean installation of {product-title} version 3.9 has swap memory enabled,
+by having the `openshift_disable_swap` setting in the Ansible install file set
+to `false` by default.
+
+If you had enabled swap memory, then the upgrade to {product-title} version 3.9
```

Comment:
How about replace the above lines like the following (no need to mention 3.9 fresh install):
A existing old version of cluster may have swap memory enabled (by having the `openshift_disable_swap` setting to false in the Ansible inventory file or manually enabled swap in the cluster). If that, the upgrade to .....